payments/price_feed.py

```diff
@@ -23,25 +34,25 @@ def get_price(currency):
         raise ("Failed to reach {}.".format(price_feed))
 
     try:
-        price = prices[currency]["rate"].replace(",", "")
+        price = prices[ticker][value_attribute]
         return price
 
     except:
         print("Failed to find currency {} from {}.".format(currency, price_feed))
         return None
 
 
-def get_btc_value(dollar_value, currency):
+def get_btc_value(base_amount, currency):
     price = get_price(currency)
-    if price is not None:
-
+    
+    if price:
```

Comment:
This will throw `Failed to get fiat value.` if the price is 0. 
```
if price is not None:
```
is better and maybe handle the edge case when price is 0 with another exception

Reply:
Fixed: https://github.com/ingolevin/SatSale/commit/3b206727eb4427d59c15a45d9ffafecc0eaab963